src/test/java/uk/gov/register/functional/LoadSerializedFunctionalTest.java

```diff
@@ -155,17 +155,30 @@ public void shouldReturnBadRequestForRegisterDefinitionWhenBeforeFieldDefinition
         Response r = send(input);
 
         assertThat(r.getStatus(), equalTo(400));
-        assertThat(r.readEntity(String.class), equalTo("{\"success\":false,\"message\":\"Exception when executing command: RegisterCommand{commandName='append-entry', arguments=[system, register:address, 2017-06-06T09:54:11Z, sha-256:8d824e2afa57f1a71980237341b0c75d61fdc5c32e52d91e64c6fc3c6265ae63]}\",\"details\":\"Field undefined: register - address\"}"));
+        assertThat(r.readEntity(String.class), equalTo("{\"success\":false,\"message\":\"Exception when executing command: RegisterCommand{commandName='append-entry', arguments=[system, register:address, 2017-06-06T09:54:11Z, sha-256:b9f80885b11cbc9064970214387571ebfae6795f62bd79723163a3a91162537e]}\",\"details\":\"Field undefined: register - address\"}"));
     }
     
     @Test
-    public void shouldReturnBadRequestForSystemEntryAfterUserEntry() throws IOException {
-        String input = new String(Files.readAllBytes(Paths.get("src/test/resources/fixtures/serialized", "register-by-registry-invalid-entry-ordering.rsf")));
+    public void shouldAllowRegisterTextToBeUpdatedAfterUserEntry() throws IOException {
+        String input = new String(Files.readAllBytes(Paths.get("src/test/resources/fixtures/serialized", "register-by-registry-valid-entry-ordering.rsf")));
         
         Response r = send(input);
         
+        assertThat(r.getStatus(), equalTo(200));
+        Response response = register.getRequest(testRegister, "/register.json");
+        Map registerResourceMapFromRegisterRegister = response.readEntity(Map.class);
+        Map<?, ?> registerRecordMapFromRegisterRegister = (Map) registerResourceMapFromRegisterRegister.get("register-record");
+        assertThat(registerRecordMapFromRegisterRegister.get("text"), equalTo("Register of registers X"));
+    }
+
+    @Test
+    public void shouldNotAllowRegisterNonTextValuesToBeUpdatedAfterUserEntry() throws IOException {
+        String input = new String(Files.readAllBytes(Paths.get("src/test/resources/fixtures/serialized", "register-by-registry-invalid-entry-ordering.rsf")));
+
+        Response r = send(input);
+
         assertThat(r.getStatus(), equalTo(400));
-        assertThat(r.readEntity(String.class), equalTo("{\"success\":false,\"message\":\"System entries must be added before user entries (line: 19): RegisterCommand{commandName='append-entry', arguments=[system, register:register, 2017-06-06T09:54:11Z, sha-256:f202653182ebd63c34177a7176ce77f1aae63482ce6914ec1c71683a6d7131ab]}\"}"));
+        assertThat(r.readEntity(String.class), containsString("Definition of register register does not match Register Register"));
```

Comment:
We should probably be consistent with the other tests here in what we're checking, for example `shouldReturnBadRequestWhenRegisterIsMissingFieldsDefinedInEnvironment`